data/presets/fields/surveillance/type.json

```diff
@@ -0,0 +1,8 @@
+{
+    "key": "surveillance:type",
```

Comment:
Let's skip `surveillance:type`.. 98% of the values are just "camera", and the other values are noisy.